### ISSUE TITLE
Ajax requests for view filters are failing on 2.7

### ISSUE THREAD

**Issue by camikazegreen** [az-digital/az_quickstart#2499]
## Problem/Motivation
Any interactive filters that require ajax requests are failing. 

### Describe the bug
This message is posted in the console:
```
Object { message: '\nAn AJAX HTTP error occurred.\nHTTP Result Code: 400\nDebugging information follows.\nPath: /views/ajax\nStatusText: Bad Request\nResponseText: \nform.antibot * :not(.antibot-message) { display: none !important; }\nClient error | Arizona Quickstart\nSkip to main content\nToolbar items\nBack to site\nManage\nAdministration menuContentStructureAppearanceExtendConfigurationPeopleReportsHelp      \nShortcuts\nUser-defined shortcutsAdd contentAll contentEdit shortcuts      \nEdit\nTour\ncamerong\nUser account actionsView profileEdit profileLog out      \nQuickstart 2.7.0-rc1\nQuickstart linksSettingsReleasesDocumentationAnalytics Dashboard      \npre-release (Pantheon)\nEnvironment switcherConfigure      \nGo to\nSearch\nsearch\nsearch \nsearch \nmenu \nmenu \nQuickstart 1\nArizona Digital\nRequest Arizona Site\nI am a\nSelect your audience\nchoose an option\nContent Editor\nContent Administrator\nDeveloper\nGo\nto the page for that group\nLive Training\nhome \nhome \nclose \nclose \nSearch\nsearch\nGet Started\nOverview\nLogging In\nAdding Content\nFinding Content\nLorem Ipsum\nCreate Content\nEditor Toolbar\nAdding Pages\nAdding Courses\nAdding Events Pages\nAdding Publications\nAdding News Pages\nAdding Carousel Items\nPeople &amp; Directories\nImages, Videos, and Files\nCreating and Using Forms\nDuplicating Content\nTranslating Content\nPage Elements\nText\nAccordions\nCards\nContacts\nPhoto Galleries\nSplit Screen\nText with Background\nText on Media\nViews\nReorder Page Elements\nSite Admin\nManaging Users\nManaging Menus\nTaxonomy\nBlocks\nRedirects\nContent Moderation &amp; Review\nAdding Views\nAnalytics\nSite Branding\nLogo &amp; Header Configuration\nGlobal Footer\nSocial Media Block\nSEO &amp; Analytics\nBest Practices\nAccessibility Guidelines\nUsing Headings\nDescriptive Links\nWriting for the Web\nBreadcrumbs\nUpdates\nHome\nSystem\nClient error\nClient error\nA client error happened\nEmployment\nEmergency Information\nTitle IX / Non-Discrimination\nCampus Safety\nUAlert\nCopyright\nCampus Accessibility\nContact Us\nFeedback\nInformation For\nFuture Students\nCurrent Students\nFaculty &amp; Staff\nAlumni &amp; Donors\nParents &amp; Visitors\nCorporations &amp; Businesses\nTopics\nAbout the University\nAcademics\nArts &amp; Museums\nAthletics &amp; Recreation\nColleges, Schools, Departments\nDiversity\nEnvironment &amp; Sustainability\nInternational Engagement\nHealth &amp; Medical\nLibraries\nOutreach &amp; Extension\nResearch &amp; Innovation\nPurpose, Mission &amp; Values\nResources\nA-Z Index\nCalendars\nCampus Map\nNews\nPhonebook\nWeather\nConnect\nFacebook\nTwitter\nInstagram\nLinkedIn\nYouTube\nWe respectfully acknowledge the University of Arizona is on the land and territories of Indigenous peoples. Today, Arizona is home to 22 federally recognized tribes, with Tucson being home to the O’odham and the Yaqui. Committed to diversity and inclusion, the University strives to build sustainable relationships with sovereign Native Nations and Indigenous communities through education offerings, partnerships, and community service.\nUniversity Information Security and Privacy\n&copy; 2023 The Arizona Board of Regents on behalf of The University of Arizona.\nMasquerade\nUsername\nSwitch\n{"ajaxPageState":{"theme":"az_barrio","libraries":"az_barrio\\/az-barrio-off-canvas-nav,az_barrio\\/global-styling,az_card\\/az_card_title_hover,az_flexible_page\\/az_flexible_page_theme,az_paragraphs\\/az_paragraphs.az_cards,az_paragraphs\\/az_paragraphs.az_text,az_select_menu\\/az_select_menu,better_exposed_filters\\/auto_submit,better_exposed_filters\\/general,blazy\\/bio.ajax,bootstrap_barrio\\/form,bootstrap_barrio\\/global-styling,bootstrap_barrio\\/links,bootstrap_barrio\\/node,bootstrap_barrio\\/tabs,coffee\\/drupal.coffee,contextual\\/drupal.contextual-links,contextual\\/drupal.contextual-toolbar,core\\/drupal.active-link,environment_indicator\\/drupal.environment_indicator,google_tag\\/gtag,google_tag\\/gtag.ajax,google_tag\\/gtm,history\\/mark-as-read,paragraphs\\/drupal.paragraphs.unpublished,shortcut\\/drupal.shortcut,system\\/base,toolbar\\/toolbar,toolbar\\/toolbar.escapeAdmin,tour\\/tour,user\\/drupal.user.icons,views\\/views.ajax,views\\/views.module"},"coffee":{"dataPath":"\\/admin\\/coffee\\/get-data","maxResults":7},"gtm":{"tagIds":["GTM-ML2BZB"],"settings":{"data_layer":"dataLayer","include_classes":false,"allowlist_classes":"google\\nnonGooglePixels\\nnonGoogleScripts\\nnonGoogleIframes","blocklist_classes":"customScripts\\ncustomPixels","include_environment":false,"environment_id":"","environment_token":""}},"gtag":{"tagId":"","otherIds":[],"events":[],"additionalConfigInfo":[]},"ajaxTrustedUrl":{"form_action_p_pvdeGsVG5zNF_XLGPTvYSKCf43t8qZYSwcfZl2uzM":true,"\\/search\\/node":true},"azSelectMenu":{"ids":["az-select-menu-select-menu-form"]},"toolbar":{"breakpoints":{"toolbar.narrow":"only screen and (min-width: 16.5em)","toolbar.standard":"only screen and (min-width: 38.125em)","toolbar.wide":"only screen and (min-width: 61em)"},"subtreesHash":"L8afhJBNadxGxA0BE1cI0o8OLOXME1Gthfq7TsB_cNM"},"environmentIndicator":{"name":"pre-release (Pantheon)","fgColor":"#ffffff","bgColor":"#1e5288","addFavicon":true},"path":{"baseUrl":"\\/","pathPrefix":"","currentPath":"views\\/ajax","currentPathIsAdmin":false,"isFront":false,"currentLanguage":"en","currentQuery":{"category":"13","viewsreference":{"data":{"title":"0","offset":"","pager":"","argument":"","limit":""},"enabled_settings":{"limit":"limit","argument":"argument","title":"title","offset":"offset","pager":"pager"}}}},"pluralDelimiter":"\\u0003","user":{"uid":"2","permissionsHash":"e2bb3dc8d811d018cd1d307fe33cc4b32e5f6b64db2100c6f7686a2fb55f8a97"}}\n', name: "AjaxError", stack: "@https://pre-release-azs-quickstart.pantheonsite.io/core/misc/ajax.js?v=10.0.9:184:32\n@https://pre-release-azs-quickstart.pantheonsite.io/core/misc/ajax.js?v=10.0.9:1798:3\n" }
```

### To Reproduce
Here area few examples of pages that are experiencing this issue:
- https://pre-release-azs-quickstart.pantheonsite.io/live-training (The Today, This Week, This Month filters. The widget updates, but the results do not. An ajax error is presented in the console.)
- https://pre-release-azs-quickstart.pantheonsite.io/updates (The Filter by Category option does not filter the results. An ajax error is presented in the console.)

### Desktop (please complete the following information):
 - OS: Mac OS 
 - Browser Firefox 
 - Version 113.0.2

**Comment by camikazegreen** [comment]
This also happens when attempting to remove instances from an Event.

**Comment by joeparsons** [comment]
https://www.drupal.org/project/viewsreference/issues/3326841

**Comment by bberndt-uaz** [comment]
The patch for the viewsreference module added by PR #2505 appears to apply on new installs but it may not apply correctly on existing Quickstart sites.

**Comment by bberndt-uaz** [comment]
Closing this issue again since the patch in the PR **does** resolve this problem. We noticed this issue in a pre-release test site for Quickstart 2.7-rc2, but determined that an issue with composer resulted in the patch not being applied. We were able to ensure the patch was applied by adding `"enable-patching": true` in the "extra" section of the upstream composer.json (not the root composer.json).
